packages/@sanity/code-input/src/groq.js

```diff
@@ -0,0 +1,611 @@
+/* eslint-disable no-undef */
+const rules = {
```

Comment:
Maybe add a comment to signal where this definition is coming from, to make it easier to keep in sync?

```suggestion
// Grammar from https://github.com/sanity-io/vscode-sanity
const rules = {
```